src/components/ModelingMachineProvider.tsx

```diff
@@ -1037,6 +1038,21 @@ export const ModelingMachineProvider = ({
     modelingSend({ type: 'Delete selection' })
   })
 
+  // Allow ctrl+alt+c to center to selection
+  useHotkeys(['ctrl + alt + c'], () => {
```

Comment:
`mod` gets translated to Control on Windows and Linux and Command on Mac.

```suggestion
  useHotkeys(['mod + alt + c'], () => {
```